.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
why all these versions? This will be a compiled lib (as the design-system)

Reply:
We use this lib inside the console that use node 16. I wanted to make sure it went everywhere

Reply:
I agree with what you say. 
However, I think it is still useful for us who use vine libraries or webpacks than for end users.
Basically this lib is designed for node 16. 
The matrix of test environments allows us to see into the future

Reply:
then we would want to use lts (20) and latest..

Reply:
yes in fact we put node 18, and 20. this repo is meant to run on the console and so node 16